src/MSIExtract/App.xaml.cs

```diff
@@ -26,7 +26,7 @@ protected override void OnStartup(StartupEventArgs e)
         {
             base.OnStartup(e);
 
-            if (!CheckPackageIdentity())
+            if (!CheckPackageIdentity() && !System.Diagnostics.Debugger.IsAttached)
```

Comment:
I was not sure about the purpose of this check, since it seems to run fine outside the package?
Since I was not sure of the purpose, I did not disable the code entirely, just made debugging somewhat easier.